### DISCUSSION TITLE
Open documents as overlay or new tab

### DISCUSSION THREAD

**Discussion by patsch9** [jonaswinkler/paperless-ng#697] [Archived FRs]
Hey,

when i click on the view button the document opens in the same tab as paperless-ng. It would be nice when the documents open as a overlay to the actual tab or as a new tab.

Thank you! Great Work!

**Comment by jonaswinkler** [comment]
I think we had this discussion somewhere before, but I can't find it right now. The gist of it was:

* Opening tabs by default is really bad If a user does not want that to happen, because they cannot prevent it.
* Users who do want to open things in new tabs can do so by using the respective menu items of their browser. Most browsers even have shortcuts for that (Example: Chrome opens in a new window on shift+click).

**Reply by patsch9** [reply]
Hi thanks for the answer. Is it possible to implement something where i can choose between new tab or same tab?

It would be even nicer when the document open as an overlay to the actual window and then you can push on exit or x to close it without going back to the last site with a reload.

**Reply by shamoon** [reply]
I think #487 is where we discussed this and PR #628 adds pop up previews

And I agree about the link not defaulting to new tab, nor does it seem worth an entire setting when the solution is just to hold down one keyboard modifier key if you do want a new tab, and defaulting to new window can make mobile use kinda frustrating. This is sorta the default on the web, after all
